public/app/app-init.js

```
@@ -1,644 +1,414 @@
-
-    liveStream = await navigator.mediaDevices.getUserMedia({
-      audio: {
-        sampleRate:       48000,
-        channelCount:     2,
-        echoCancellation: false,  // we handle this ourselves
-        noiseSuppression: false,  // same
-        autoGainControl:  false,
-      },
-    });
+    const arrayBuffer = await file.arrayBuffer();
+    const tmpCtx = new AudioContext();
```

ESLint

[error] 'AudioContext' is not defined.

(no-undef)